app/controllers/admin/schools/induction_coordinators_controller.rb

```diff
@@ -31,8 +54,36 @@ def set_school
       @school = School.find params[:school_id]
     end
 
-    def form_params
-      params.require(:nominate_induction_tutor_form).permit(:full_name, :email)
+    def school_induction_tutor
+      @school.induction_coordinators.first
+    end
+
+    def school_induction_tutor_attributes
+      user = @school.induction_coordinators.first
```

Comment:
`school_induction_tutor`?

Reply:
I can change it but isn't that how they are referred to in the UI?

Reply:
Oh, I meant that we could reuse the method above to get the user here